src/handlers/web3.js

```diff
@@ -213,7 +269,9 @@ export const getTxDetails = async transaction => {
   const { to } = transaction;
   const data = transaction.data ? transaction.data : '0x';
   const value = transaction.amount ? toHex(toWei(transaction.amount)) : '0x00';
-  const gasLimit = toHex(transaction.gasLimit) || undefined;
+  const gasLimit = transaction.gasLimit
+    ? toHex(transaction.gasLimit) || undefined
+    : undefined;
```

Comment:
hm am I missing something here? this looks strange

Reply:
can just remove `|| undefined` in line 273 `toHex(transaction.gasLimit) || undefined`